packages/SwingSet/docs/dynamic-vats.md

```diff
@@ -1,14 +1,14 @@
 # Dynamic Vats
 
-A SwingSet machine has two sets of vats. The first are the *Static Vats*, which are defined in the configuration object, and created at startup time. The root objects of these vats are made available to the `bootstrap()` method, which can wire them together as it sees fit.
+A SwingSet machine has two sets of vats. The first are the _Static Vats_, which are defined in the configuration object, and created at startup time. The root objects of these vats are made available to the `bootstrap()` method, which can wire them together as it sees fit.
```

Comment:
I can't speak for @warner on the value of pretty swingset docs formatting vs. git history churn.

Reply:
re: cost of churn, git blame will show `docs: prettier` and in the Github UI one can click the "earlier" icon to see the human editor.